wagtail/core/collectors.py

```diff
@@ -129,11 +231,37 @@ def block_tree_paths(self, block, ancestors=()):
             yield ancestors + (block,)
 
     def find_block_type(self, block_types):
+        """
+        Yields all the possible paths for ``Block`` instances of one of the
+        ``Block`` subclasses passed as the ``block_types``.
+
+        Example:
+
+        >>> from wagtail.core.blocks import CharBlock, TextBlock
+        >>> from wagtail.core.fields import StreamField
+        >>> stream_field = StreamField([
+        ...     ('title', CharBlock()),
+        ...     ('items', ListBlock(TextBlock())),
+        ... ])
+        >>> [[b.__class__.__name__ for b in path]
+        ...  for path in StreamFieldCollector(stream_field)
+        ...              .find_block_type((CharBlock, TextBlock))]
+        [['StreamBlock', 'ListBlock', 'TextBlock']]
```

Comment:
Shouldn't this be `[['StreamBlock', 'CharBlock'], ['StreamBlock', 'ListBlock', 'TextBlock']]`?